boolean_parser/mixins/sqla.py

```diff
@@ -177,13 +233,31 @@ def _filter_one(self, model, field=None, condition=None):
                 # x=*5 -> x LIKE '%5' (x ends with 5)
                 field = getattr(model, self.name)
                 value = self.value
-                if value.find('*') >= 0:
+                if value.lower() == 'null':
+                    condition = field.is_(None)
+                elif value.find('*') >= 0:
                     value = value.replace('*', '%')
-                    condition = field.ilike(
-                        bindparam(self.bindname, value))
+                    condition = field.ilike(value)
+                else:
+                    condition = field.ilike('%' + value + '%')
+            elif isinstance(field.type, sqltypes.BOOLEAN) or \
```

Comment:
I use `bindparam` to map the value to the full "table.parameter" name.  This makes the internal sql statement syntax a bit more readable, especially when one queries on the same column name from different tables.  This is what `lower_value` is from the output of `bind_and_lower_value` at line 147 does.  

I realized this bit of code never got properly translated and the error you were seeing with `self.bindname` was from old code that no longer exists.  If possible, I'd like to keep the `bindparam` functionality for these cases.  You could use the `self.fullname` instead, and use something like `condition = lower_field.ilike(bindparam(self.fullname, value))` at line 240, and similarly for line 242.

Reply:
And similarly in the other code blocks where `ilike` is used.

Reply:
I've added the `bindparam` functionality. Thanks for the pointers.